### DISCUSSION TITLE
Revisiting 2011 Feature Request for Date Filtering in Etsy API Listings Endpoints

### DISCUSSION THREAD

**Discussion by Mparker25** [etsy/open-api#1170] [Ideas]
I'm revisiting a [feature request from 2011 regarding the need for a date filtering option](https://groups.google.com/g/etsy-api-v2/c/r37pfCintnQ/m/022ju8EGAgAJ) in the listings endpoints of the Etsy API. 

Currently, when polling for listings, we are required to pull all listings for each customer, which proves to be a time and resource-intensive process, especially when dealing with multiple Etsy accounts. To optimize our approach, we have separated the task of pulling images for only the most recent listings.

I'm reaching out to see if there is a possibility of adding a filter `by date` for listings. This enhancement would significantly reduce the number of API calls and resources needed, making the process more efficient and streamlined

**Comment by Gareth064** [comment]
Hey, now that you have raised this discussion here, I'd recommend you email Etsy at developer@etsy.com to ask them to look at it.

The don't watch this discussion repo to see people asking for things etc.

**Reply by Mparker25** [reply]
That’s unfortunate. Thanks for the info. I’ll try to update the thread should I get a response!
